src/main/resources/reference.conf

```diff
@@ -9,34 +10,11 @@ publication {
   # and specify it's filename and password here
   server.truststore.location = ""
   server.truststore.password = ""
-  spray.can.server {
-    ssl-encryption = "off"
-    request-timeout = 15s
-    pipelining-limit = 128
-    request-chunk-aggregation-limit = "30m"
-    parsing.max-content-length = "30m"
-  }
 }
 
 server.address="::0"
 rrdp.port = 7788
 
-spray.can.host-connector {
-  # The maximum number of parallel connections that an `HttpHostConnector`
-  # is allowed to establish to a host. Must be greater than zero.
-  max-connections = 1024
-
-  # If this setting is enabled, the `HttpHostConnector` pipelines requests
-  # across connections, otherwise only one single request can be "open"
-  # on a particular HTTP connection.
-  pipelining = on
```

Comment:
And maybe this one as well.

Reply:
My understanding is that HTTP pipelining queues the requests (c.f. executes them in parallel). It is not commonly used and I could not find whether the client we use supports it.

I would propose to skip it since the current release does not have it and the Akka HTTP [docs](https://doc.akka.io/docs/akka-http/current/server-side/low-level-api.html#controlling-server-parallelism) discourage it's use.